docs/UserGuide.md

```diff
@@ -146,22 +145,63 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+#### 2.5 Deleting a person : `delete`
 
-### Deleting a contact number : `delete`
+Deletes the specified person from the address book.
 
+Format: `delete INDEX`
 
-Deletes the specified person from the contact list.
+* Deletes the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
 
-Format: `delete NAME`
+Examples:
+* `list` followed by `delete 2` deletes the 2nd person in the contact list.
+* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-* Deletes the person with the specified `NAME`.
+#### 2.6 Clearing all entries : `clear`
 
-Example:
-* `delete John` deletes any entry with the name `John` from the contact list.
+Clears all entries from the contact list.
+
+Format: `clear`
+
+#### 2.7 Archiving a person : `archive`
+Archives the specified employee's contact detail from the tCheck
+
+Format: `archive INDEX`
+
+* Archives the employee at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `list` followed by `archive 2` archives the 2nd person in the employees' contact details.
+* `find Betsy` followed by `archive 1` deletes the 1st person in the results of the `find` command.
+
+#### 2.8 Archiving all employees : `archive all`
+Archives all employees' contact detail from the tCheck
+
+Format: `archive all`
+
+#### 2.9 Listing all archived employees : `archive list`
+Shows a list of all archived employees' contact details in tCheck.
+
+Format: `archive list`
 
-### Ingredients Tracking
+### 3. Commands - Ingredients Tracking <a name="commands-ingredients-tracking"></a>
 
-#### Set Ingredient Level : `set`
+#### 3.1 Setting one ingredient to initial level : `set`
+
+* Users can set the level for one type of ingredient individually.
```

Comment:
Remove the bullet points in this section to standardise the description field for the features. Instead of starting with a noun, start with a verb (once again to standardise across the UG). Eg. Set the level ...

Reply:
Removed. Thanks (: